app/models/match_decisions/homeless_set_aside/set_asides_hsa_acknowledges_receipt.rb

```diff
@@ -74,9 +74,9 @@ def acknowledged
         end
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.